### ISSUE TITLE
Colour contrast too low in postman client

### ISSUE THREAD

**Issue by glenthomas** [a85/POSTMan-Chrome-Extension#3843]
## App Details:
```
Postman for Windows
Version 5.3.2
win32 10.0.15063 / x64
```
## Issue Report:
 1. Did you encounter this recently, or has this bug always been there: Always
 2. Expected behaviour: Higher contrast
 3. Console logs: N/A
 4. Screenshots (if applicable)

The contrast in the Windows (and I assume other platforms) client is too low, it is killing my eyes. Grey on black or grey on white does not provide enough contrast between text and background. The whole application has a kind of permanently 'disabled' look to it, which it makes it painful to use for extended periods of time. The text needs to be darker in the light theme and lighter in the dark theme. Same for element borders, etc.

**Comment by connorsadlervelo** [comment]
https://github.com/postmanlabs/postman-app-support/issues/2450

**Comment by xeruf** [comment]
Wow, this is such a simple thing to fix but it still gets ignored for almost 2 years. I am sitting outside and can see _NOTHING_ in the dark theme! And even in the light theme I have to strain my eyes more than necessary.

**Comment by timbaileyjones** [comment]
@MrGlenThomas --- constrast is terrible on MacOS too.

I'm INSIDE on a 43-inch monitor and still cannot see the `</>` (Code) icon without MacOS's zoom/accessibility feature.  
Light gray on white, or dark gray on black, depending on theme.  

How about adding a "high-contrast theme", next to the light / dark ones?

**Comment by SamSoldatenko** [comment]
I SO HATE POSTMAN DEVELOPERS!!! YOU STUPID MORONS!

**Comment by SamSoldatenko** [comment]
USE F'king black colort!!! It is `#000000`

**Comment by SamSoldatenko** [comment]
THIS ISSUE IS 5 YEARS!!!
Stop killing my eyes!

**Comment by mihmig** [comment]
Let's look for alternatives.
1. Insomnia
2. HttPie (Desktop version recently released)
3. Built in Http-client in Intellij products (my choice at the moment, but it does not completely satisfy me)

**Comment by chris-torruella** [comment]
Hey everyone, we heard you and I'm excited to share that Postman v11 introduces a whole set of application themes accessible via Settings/Themes. Now you can also sync these with your System preferences. But very importantly, we've also included High Contrast themes for enhanced contrast as needed. We hope these will help to elevate your experience, and we're eager to hear your feedback. Cheers!
